src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1832,25 +1844,28 @@ private XMPPError removeSources(
 
         logger.info("Removing sources from " + participantJid + ": " + removedSources);
 
-        participants.stream()
-            .filter(otherParticipant -> otherParticipant != participant)
-            .forEach(
-                otherParticipant ->
-                {
-                    if (otherParticipant.isSessionEstablished())
-                    {
-                        jingle.sendRemoveSourceIQ(
-                            removedSources,
-                            removedGroups,
-                            otherParticipant.getJingleSession());
-                    }
-                    else
-                    {
-                        logger.warn("Remove source: no jingle session for " + participantJid);
-                        otherParticipant.scheduleSourcesToRemove(removedSources);
-                        otherParticipant.scheduleSourceGroupsToRemove(removedGroups);
-                    }
-                });
+        if (sendSourceRemove)
```

Comment:
Maybe negate this (& return) and move it above the logger statement?

Reply:
I know this would save some indentation, but I prefer it like this. If we add more code to the function, the early `return` makes it easier to do something unindented.

Reply:
e.g. this would probably be a bug:
```
if (!sendSourceRemove)
    return null;
    
 sendSourceRemove();
 doSomethingElse());   
 return null;
```